### ISSUE TITLE
platform-aware warnings (no-member error for WSAEWOULDBLOCK)

### ISSUE THREAD

**Issue by vlovich** [PyCQA/pylint#2792]
### Steps to reproduce
1. test.py:
```python
import errno
import sys


if sys.platform == "win32":
    errno.WSAEWOULDBLOCK
```
2. pylint -E test.py

### Current behavior
When run on OSX:
************* Module test
test.py:2:0: E1101: Module 'errno' has no 'WSAEWOULDBLOCK' member (no-member)

### Expected behavior
No error since WSAEWOULDBLOCK is valid on win32 & not valid otherwise.

I understand that pylint doesn't yet support control flow.

### pylint --version output
pylint 2.3.1
astroid 2.2.3
Python 3.7.2 (default, Feb 13 2019, 15:31:10) 
[Clang 10.0.0 (clang-1000.11.45.5)]

**Comment by jenstroeger** [comment]
I’m running into a similar issue with platform specific exit codes (e.g. [`os.EX_OK`](https://docs.python.org/3/library/os.html#os.EX_OK)) that [exist on UNIX systems but not on Windows](https://docs.python.org/3/library/intro.html#notes-on-availability).

For example, it would be great to disable `pylint` warnings for a certain platform, e.g.
```python
    if sys.platform in ("linux", "darwin"):
        EX_OK = os.EX_OK  # pylint: disable[win32]=no-member
    elif sys.platform == "win32":
        EX_OK = 0
    else:
        raise RuntimeError(f"Unsupported host platform: {sys.platform}")
```

**Comment by Pierre-Sassoulas** [comment]
In the past we've created brains for astroid combining attribute from all platforms (sometime the lib is built differently on each platfrm so there's no other way).

**Comment by jenstroeger** [comment]
@Pierre-Sassoulas, are you suggesting to open an issue with [`astroid`](https://github.com/PyCQA/astroid) instead of here with `pylint`?

**Comment by Pierre-Sassoulas** [comment]
Yes, or a MR directly. Sorry for the delay the MR I was referencing was surprisingly hard to find 😄 [Creating a brain with a mashup of multiple OS's code](https://github.com/PyCQA/astroid/pull/1172)

**Comment by Pierre-Sassoulas** [comment]
I can't reproduce on linux, could someone with a MacOs try to reproduce please ? :)

**Comment by DanielNoord** [comment]
No longer reproduces!
